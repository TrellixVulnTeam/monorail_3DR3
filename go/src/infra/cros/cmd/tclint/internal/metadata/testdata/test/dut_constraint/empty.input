{
    "remoteTestDrivers": [
        {
            "name": "remoteTestDrivers/tauto",
            "tests": [
                {
                    "name": "remoteTestDrivers/tauto/tests/foo",
                    "dut_constraint": {}
                }
            ]
        }
    ]
}