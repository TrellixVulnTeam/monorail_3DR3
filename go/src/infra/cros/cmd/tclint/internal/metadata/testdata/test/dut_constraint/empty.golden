[
"## Errors ##",
"Test 'remoteTestDrivers/tauto/tests/foo': dut_constraint: some constraint must be set"
]